src/admin/includes/formDefinitions/callbacks.php

```diff
@@ -0,0 +1,24 @@
+<?php
+function processInsert(){
+  $localvars = localvars::getInstance();
+  $formInfo  = $_POST['MYSQL'];
+
+  $roomID = isset($formInfo['roomID']) ? $formInfo['roomID'] : NULL;
```

Comment:
remove this line of code. 

if $formInfo is assigned the value of $_POST['MYSQL'];
and if $roomID is assigned the value of $formInfo['roomID'];
then $roomID = $_POST['MYSQL']['roomID']

just use $_POST['MYSQL']['roomID'] and save the memory. The web server (and infrastructure team) will thank you.